packages/tpetra/core/example/advanced/Benchmarks/CrsMatrixMatVec.cpp

```diff
@@ -346,23 +356,29 @@ main (int argc, char* argv[])
     // (or even denorms) via traps.  This is very expensive, so if the
     // norms increase or decrease a lot, that might trigger the slow
     // case.
-    const SC X_val = static_cast<SC> (1.0) /
-      static_cast<SC> (opts.numEntPerRow);
-    X.putScalar (X_val);
-    Y.putScalar (0.0);
+    timer = TimeMonitor::getNewCounter ("Tpetra CrsMatrix Benchmark: create vectors");
+    {
+      TimeMonitor timeMon (*timer);
+      const SC X_val = static_cast<SC> (1.0) /
+        static_cast<SC> (opts.numEntPerRow);
```

Comment:
This might fail if SC is a complex number, since there's no conversion defined from integers to `std::complex<T>`.  However, I'm pretty sure you mean SC to be `double` here, so this should be harmless.

Reply:
Yes that a good point, I think that something like `static_cast<SC>(1.0 / opts.numEntPerRow);` makes more sense and would be safer, I'll push that in a subsequent PR.